service/internal/cache/cache.go

```diff
@@ -0,0 +1,37 @@
+package cache
+
+import (
+	"time"
+
+	gocache "github.com/patrickmn/go-cache"
+)
+
+type Float64Cache struct {
+	underlying *gocache.Cache
+}
+
+func NewFloat64Cache(expiration time.Duration) *Float64Cache {
+	c := &Float64Cache{
+		underlying: gocache.New(expiration, expiration/2),
```

Comment:
Since I guess this is inspired by `operatorkit` can Mr. @kopiczko maybe tell us why the given value is halved?